Benchmark `ffmpeg_ffmpeg_demuxer_fuzzer` fails to build.

## `ffmpeg_ffmpeg_demuxer_fuzzer`
> XEVI.c:150:16: error: implicit declaration of function 'reallocarray' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        temp_visual = Xmallocarray(n_visual, sz_VisualID32);
                      ^
./reallocarray.h:44:32: note: expanded from macro 'Xmallocarray'
#define Xmallocarray(n, size)           Xreallocarray(NULL, (n), (size))
                                        ^
./reallocarray.h:39:5: note: expanded from macro 'Xreallocarray'
    reallocarray((ptr), ((n) == 0 ? 1 : (n)), size)
    ^
XEVI.c:150:14: warning: incompatible integer to pointer conversion assigning to 'VisualID32 *' (aka 'unsigned int *') from 'int' [-Wint-conversion]
        temp_visual = Xmallocarray(n_visual, sz_VisualID32);

This was caused by a commit to a dependency (`libXert`) and was temporarily fixed by reverting to one commit earlier (d965a1a8ce9331d2aaf1c697a29455ad55171b36) in #1419.
Now that they pushed another commit in `libXert` and still did not fix this error.

Given this benchmark is not used in experiments, maybe we could remove it (from CI tests)?

--- a/benchmarks/ffmpeg_ffmpeg_demuxer_fuzzer/Dockerfile
+++ b/benchmarks/ffmpeg_ffmpeg_demuxer_fuzzer/Dockerfile
@@ -27,11 +27,13 @@ RUN git clone https://git.ffmpeg.org/ffmpeg.git ffmpeg
 
 RUN wget https://www.alsa-project.org/files/pub/lib/alsa-lib-1.1.0.tar.bz2
 RUN git clone -n https://gitlab.freedesktop.org/mesa/drm.git
-RUN cd drm; git checkout 5db0f7692d1fdf05f9f6c0c02ffa5a5f4379c1f3 
+RUN cd drm; git checkout 5db0f7692d1fdf05f9f6c0c02ffa5a5f4379c1f3
 RUN git clone --depth 1 https://github.com/mstorsjo/fdk-aac.git
 ADD https://sourceforge.net/projects/lame/files/latest/download lame.tar.gz
-RUN git clone --depth 2 git://anongit.freedesktop.org/xorg/lib/libXext
-RUN (cd /src/libXext; git checkout d965a1a8ce9331d2aaf1c697a29455ad55171b36)
+RUN git clone git://anongit.freedesktop.org/xorg/lib/libXext && \
+    cd /src/libXext && \
+    git checkout d965a1a8ce9331d2aaf1c697a29455ad55171b36
+
 RUN git clone -n git://anongit.freedesktop.org/git/xorg/lib/libXfixes
 RUN cd libXfixes; git checkout 174a94975af710247719310cfc53bd13e1f3b44d
 RUN git clone --depth 1 https://github.com/intel/libva

